### PR TITLE
frontend release fixes (DEV-7352)

### PR DESCRIPTION
**Pull request by peter-horvath**

### CODE REVIEW

File: app/scripts/controllers/next-release.ts [hunk 1]
```diff
@@ -12,10 +12,13 @@ module labsFrontendApp
     export class NextReleaseCtrl
     {
         // @ngInject
-        constructor (private $scope: INextReleaseScope, private releases: GetAllPublicReleases)
+        constructor (private $scope: INextReleaseScope, private cachedReleases: GetAllPublicReleases)
```
Comment (luos) [inline app/scripts/controllers/next-release.ts:15]: I don't think you should care if it's cached or not in the naming :)

Comment (luos) [inline app/scripts/controllers/next-release.ts]: Ah but I see it's different to releases

Comment (peter-horvath) [inline app/scripts/controllers/next-release.ts:15]: I needed to have a different variable name, injection works based on variable names here :S

Comment (luos) [inline app/scripts/controllers/next-release.ts:15]: Ah, I see

**Pull request merged**